tests/nim-groth16-verifier/verifier_test.nim

```diff
@@ -0,0 +1,17 @@
+import
```

Comment:
You've accidentally committed the compiled `verifier_test` binary. It should be removed from history through interactive rebase.

Reply:
Fixed